ehrapy/anndata/anndata_ext.py

```diff
@@ -16,6 +16,7 @@
 from scipy.sparse import issparse
 
 from ehrapy import logging as logg
+from ehrapy.anndata._constants import EHRAPY_TYPE_KEY, NON_NUMERIC_ENCODED_TAG, NON_NUMERIC_TAG, NUMERIC_TAG
```

Comment:
Am I blind or where are you importing them from?

Reply:
Whoops, new file slipped my notice when commiting, added it with latest push